assets/helpers/user/userReducer.js

```diff
@@ -14,6 +14,7 @@ export type User = {
   firstName: ?string,
   lastName: ?string,
   isTestUser: ?boolean,
+  isPostDeploymentTestUser: ?boolean,
```

Comment:
I would define `isPostDeploymentTestUser` as `?: boolean` and initialize as `false` to be more explicit about the condition of the test.